src/ngraph/graph_util.cpp

```diff
@@ -642,3 +642,16 @@ std::vector<Output<Node>> ngraph::get_outputs_to(Node& src, Node& dst)
 
     return result;
 }
+
+bool ngraph::is_function_dynamic(const ngraph::Function* func)
```

Comment:
Can we use `const shared_ptr<ngraph::Function>&`. This seems to be the standard in ngraph.

Reply:
Any reason not to just move the logic here into `Function::is_dynamic`?

Reply:
@aprocter I thought we could reuse the code if we put in ngraph/graph_utils, but your are right, since is_dynamic will be attribute to function we can just move it to `Function::is_dynamic` :)